src/Tizen.NUI/src/internal/Interop/Interop.Window.cs

```diff
@@ -35,6 +35,9 @@ internal static partial class Window
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_new_Window__SWIG_0")]
             public static extern global::System.IntPtr NewWindow();
 
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_New__WithWindowData")]
+            public static extern global::System.IntPtr New(string name, string className, global::System.Runtime.InteropServices.HandleRef windowData);
```

Comment:
Could you please move this function after line 33? I think it is familiar with New(~~). not NewWindow()

Reply:
Done.